### DISCUSSION TITLE
Track languages with Accept-Language header

### DISCUSSION THREAD

**Discussion by Snazzah** [dubinc/dub#159] [Ideas]
Not too sure if this is an idea worth exploring, but services like Rebrandly have a field for languages, probably from parsing the `Accept-Language` header. Would this be a good idea to include in link stats?

![](https://get.snaz.in/7gUT1Ke.png)

**Comment by steven-tey** [comment]
Ooooh that's a great idea actually! Need to double-check if edge middleware includes an "Accept-Language" header though
